### ISSUE TITLE
How to plot masks of maskrcnn? 

### ISSUE THREAD

**Issue by leglandudu69** [pytorch/vision#2019]
Hello,

Does someone know how to plot masks of maskrcnn? In the output of maskrcnn_inference in roi_heads.py mask_logits pass through sigmoid to become mask_probs and its output_size is generally very small to correctly see anything (28*28 by default, which is defined in the roi_align parameters). I tried to binarize this mask_probs using cv2.threshold (with threshold value equals to np.median of mask_probs) then convert to polygon with cv2.findcontours and finally resize it in the image shape but the results are not good. 

Thanks

**Comment by hemasunder** [comment]
could you solve this issue? I'm facing similar issues with masks

**Comment by fmassa** [comment]
Hi,

We added a new function `draw_segmentation_masks` in https://github.com/pytorch/vision/blob/c2ab0c59f42babf9ad01aa616cd8a901daac86dd/torchvision/utils.py#L228-L252 which can be used to plot the segmentation masks of segmentation models (including Mask R-CNN).

I believe I've answered your question, and as such I'm closing this issue, but let us know if you have further questions
